source/Version.h

```diff
@@ -0,0 +1,25 @@
+/* Version.h
+Copyright (c) 2022 by warp-core
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef VERSION_H_
+#define VERSION_H_
+
+#include <string>
+
+// A class to get a string with the game version and
+// the latest commit hash for non-release version.
+class Version {
+public:
+	static const std::string GetString();
```

Comment:
Avoid a copy by returning a reference

Reply:
One of the possible return values is a temporary variable, which produces a warning.

Reply:
Oh bruh I'm dumb. Yeah ok then drop the `cosnt`, sry about that